core/base/inc/TNamed.h

```diff
@@ -35,7 +35,7 @@ class TNamed : public TObject {
 public:
    TNamed(): fName(), fTitle() { }
```

Comment:
While looking at it, I realized, that this could be simplified:

```suggestion
   TNamed() = default;
```

Should I just put it in the main commit and not mention it in the commit body? Or is this just "if it's not broken, don't fix it"?

Reply:
> Should I just put it in the main commit and not mention it in the commit body?

Yes you can do this if you want, as long as there are not so many modernizations unrelated to the actual change that the review becomes very hard because you don't see the forest for the trees I'm fine with that!

Reply:
done.

also mentioned it in the commit body briefly.